pages/users-page.ts

```diff
@@ -0,0 +1,42 @@
+// Users page before installation
+
+export class UsersPage {
+  readonly page: Page;
+  readonly defineUserButton: Locator;
+  readonly setRootPasswordDots: Locator;
+  readonly setRootPasswordMenuItem: Locator;
+  readonly backButton: Locator;
+  readonly noUserDefinedText: Locator;
+  readonly rootPasswordNotSetText: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.defineUserButton = page.getByRole('button', { name: 'Define a user now' });
+    this.setRootPasswordDots =  page.getByRole('row', { name: 'Password Not set Actions' }).getByRole('button', { name: 'Actions' });
+    this.setRootPasswordMenuItem =  page.getByRole('menuitem', { name: 'Set' });
+    this.backButton = page.getByRole('button', { name: 'Back' });
+    this.noUserDefinedText = page.getByText('No user defined yet');
+    this.rootPasswordNotSetText = page.getByRole('row', { name: 'Password Not set Actions' }).getByRole('gridcell', { name: 'Not set' });
+  }
+
+  async noUserDefined() {
```

Comment:
we are missing a verb, this part is new, so I would recommend...something like:
`expectNoUserDefined` so all our wrapped functions would have `expect` seems the easy way until finding more complicated cases...